src/sparse/impl/KokkosSparse_partitioning_impl.hpp

```diff
@@ -74,535 +74,6 @@ struct IotaFunctor
   View v;
 };
 
-template <typename HandleType, typename lno_row_view_t, typename lno_nnz_view_t>
-struct RCM
```

Comment:
Can't recall where the RCM, CMK and min degree went?

Reply:
@lucbv The RCM, CM and BFS was broken and not used by cluster Gauss-Seidel anymore so I took it out (Chris S tried it for something in MueLu and found it produces the wrong ordering for openmp).